spec/factories/user.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
```

Comment:
Here you would want to declare the `user` alias as:
`factory :user, aliases: [:author] do`

[OPTIONAL]